core/Makefile

```diff
@@ -69,7 +69,7 @@ ifeq (true, $(EHSM_DEFAULT_DOMAIN_KEY_FALLBACK))
 endif
 
 App_Cpp_Flags := $(App_C_Flags) -std=c++11
-App_Link_Flags := $(SGX_COMMON_CFLAGS) -L$(SGX_LIBRARY_PATH) -l$(Urts_Library_Name) -L. -lsgx_ukey_exchange -lpthread -ljsoncpp
+App_Link_Flags := $(SGX_COMMON_CFLAGS) -L$(SGX_LIBRARY_PATH) -l$(Urts_Library_Name) -L. -lsgx_ukey_exchange -lpthread -ljsoncpp -luuid
```

Comment:
is it added to Docker file? apt install uuid-dev or others?

Reply:
yes,Docker file will be add apt install uuid-dev.